server/protocols/webadmin-cli/src/main/java/org/apache/james/Picocli/Picocli.java

```diff
@@ -0,0 +1,23 @@
+package org.apache.james.Picocli;
+
+import picocli.CommandLine;
+
+@CommandLine.Command(
+        name = "james-cli",
+        description = "James Webadmin CLI"
+)
+public class Picocli implements Runnable{
+
+    @CommandLine.Option(
+            names = "--help",
+            description = "Display help for commands"
+    )
+    public boolean isNeedHelp;
+
+    @Override
+    public void run() {
+        final String helpMessage = "There is no help for any commands";
+        if (isNeedHelp) System.out.println(helpMessage);
```

Comment:
You likely don't pass the checkstyle :-)

Try to run `mvn clean package -DskipTests` on your work

Reply:
We should not generate our own help message but let picocli generate it.

See https://picocli.info/#execute